discovery/syncer_test.go

```diff
@@ -493,6 +513,12 @@ func TestGossipSyncerApplyGossipFilter(t *testing.T) {
 				1, len(msgs))
 		}
 	}
+
+	// wait for error from goroutine
```

Comment:
Nit: comments should be proper sentences, i.e., start capitalized and punctuation.

Reply:
ok